Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -91,10 +91,23 @@ $BaseFolders.Add((Join-Path $env:SystemDrive '\inetpub\temp\IIS Temporary Compre
 $BaseFolders.Add((Join-Path $env:SystemRoot '\Microsoft.NET\Framework64\v4.0.30319\Temporary ASP.NET Files').tolower())
 $BaseFolders.Add((Join-Path $env:SystemRoot '\System32\Inetsrv').tolower())
 
-# Add all database folder paths
-foreach ($Entry in (Get-MailboxDatabase -Server $Env:COMPUTERNAME)) {
-    $BaseFolders.Add((Split-Path $Entry.EdbFilePath -Parent).tolower())
-    $BaseFolders.Add(($Entry.LogFolderPath.pathname.tolower()))
+
+$CmdAvailable = Get-Command Get-ExchangeServer -ErrorAction SilentlyContinue
+if ( -not $CmdAvailable) {
+    try {
+        Add-PSSnapin Microsoft.Exchange.Management.PowerShell.SnapIn
```

Comment:
We should not be directly adding in the PowerShell SnapIn like this, as we bypass RBAC. 

Use the `Confirm-ExchangeShell` from the Shared location.